### PR TITLE
Add few more possible values to all_options.yaml

### PR DESCRIPTION
**Pull request by andrrizzi**
Just a small update of the YAML docs. Ready to be merged.

### CODE REVIEW

File: examples/yank-yaml-cookbook/all-options.yaml [hunk 1]
```diff
@@ -27,12 +27,12 @@ options:
                                                 # with the constraint of being at a distance greater than
                                                 # randomize_ligand_close_cutoff from the receptor.
   temperature: 298.0 * kelvin                   # Temperature of the system.
-  pressure: 1.0 * atmosphere                    # Pressure of the system.
+  pressure: 1.0 * atmosphere                    # Pressure of the system. Set to null for NVT ensemble.
   hydrogen_mass: 1.0 * amu                      # Hydrogen mass for HMR simulations.
-  constraints: HBonds                           # Constrain bond lengths and angles. Possible values are None,
+  constraints: HBonds                           # Constrain bond lengths and angles. Possible values are null,
```
Comment (jchodera) [inline examples/yank-yaml-cookbook/all-options.yaml]: Are `None` and `null` interpreted differently?

Comment (andrrizzi) [inline examples/yank-yaml-cookbook/all-options.yaml]: To be more clear
```yaml
constraints: None
```
would be interpreted as the string `'None'` by YAML and `schema` will reject it.

**Comment by Lnaden** [comment]
Looks good to me

**Comment by andrrizzi** [comment]
Thanks! Merging.

**Pull request merged**